tests/test_monitoring_patterns.py

```
@@ -0,0 +1,508 @@
+                event.body = "John 3:16"
+                event.sender = "@user:matrix.org"
+                event.server_timestamp = 1234567890000  # Use milliseconds
+
+                room = MagicMock()
+                room.room_id = mock_config["matrix_room_ids"][0]  # Use configured room
+
+                # The real bot doesn't have try/catch, so exception will propagate
+                try:
+                    await bot.on_room_message(room, event)
+                except Exception:
+                    pass  # Expected exception
```

Ruff (0.12.2)

`try`-`except`-`pass` detected, consider logging the exception

(S110)